srcopsmetrics/cli.py

```diff
@@ -113,34 +109,24 @@ def cli(
         tupled_repos = [(lambda x: (x[0], x[1]))(repo.split("/")) for repo in repos]
         analyse_projects(projects=tupled_repos, is_local=is_local, entities=entities)
 
-        for repo in repos:
-            remove_previously_processed(repo)
-
-    for project in repos:
-        os.environ["PROJECT"] = project
-
-        if process_knowledge:
-            remove_previously_processed(project)
-            issues = KnowledgeStorage(is_local=is_local).load_previous_knowledge(
-                project_name=project, knowledge_type=EntityTypeEnum.ISSUE.value
-            )
-            prs = KnowledgeStorage(is_local=is_local).load_previous_knowledge(
-                project_name=project, knowledge_type=EntityTypeEnum.PULL_REQUEST.value
-            )
-            Processing(issues=issues, pull_requests=prs).regenerate()
-
-        if visualize_statistics:
-            visualize_project_results(project=project, is_local=is_local)
-
-        if reviewer_reccomender:
-            reviewer_assigner = ReviewerAssigner()
-            reviewer_assigner.evaluate_reviewers_scores(project=project, is_local=is_local)
-
-    if visualize_statistics and repository is not None:
-        visualize_project_results(project=repository, is_local=is_local)
-    elif visualize_statistics and organization is not None:
-        # TODO
-        raise NotImplementedError
+    #     for repo in repos:
```

Comment:
why this part is commented?

Reply:
Oh my bad, didn't notice I have commented out this.
Is it better now?

Reply:
I still see commented parts

Reply:
see this commit https://github.com/thoth-station/mi/pull/186/commits/1aac665e22048465390c5b13754aa710570a4dc2

Reply:
line 113 is commented indeed :)

Reply:
dang it! my evil intentions of leaving commented code just for my laziness were revealed :D !
thx man, should be all clear now